linux_os/guide/services/obsolete/r_services/no_rsh_trust_files/bash/shared.sh

```diff
@@ -1,6 +1,10 @@
 # platform = Red Hat Virtualization 4,multi_platform_ol,multi_platform_rhel
-find /home -maxdepth 2 -type f -name .rhosts -exec rm -f '{}' \;
 
-if [ -f /etc/hosts.equiv ]; then
-	/bin/rm -f /etc/hosts.equiv
-fi
+# Identify local mounts
+MOUNT_LIST=$(df --local | awk '{print $6}' | grep -v Mounted | grep -v '^/dev')
+
+# Find file on each listed mount point
+for cur_mount in ${MOUNT_LIST}
+do
+	find ${cur_mount} -xdev -type f \( -name ".rhosts" -o -name "hosts.equiv" \) -exec rm -f {} \;
+done
```

Comment:
I think it should be enough to look into the `/home` and `/root`.

Reply:
Is it? because if you look at the `/etc/passwd` not all users have their home directories only under `/home` or `/root`.

Reply:
We can probably write remediation based on the `6.2.13` (https://workbench.cisecurity.org/sections/300333/recommendations/536114) but if we would like to have similar OVAL check, wouldn't it be too complex?

Reply:
I see, you are correct, :).
So the check and remediation could look for the users home and act only in those directories.

The OVAL wouldn't be too complex, but would require defining a variable based on contents of `/etc/passwd`.

Reply:
I've looked into this and if we would try to check it for every user from `/etc/passwd` the check would be traversing the whole FS so I've decided to leave the OVAL check as it is. Just for the record, following users have `/` as their home directory on my system (most of them are services anyway):
```
nobody /
dbus /
systemd-coredump /
systemd-network /
systemd-resolve /
polkitd /
qemu /
usbmuxd /
radvd /
nm-openvpn /
nm-openconnect /
tcpdump /
systemd-timesync /
flatpak /
firebird /
```
I even found that some users like `rtkit` have `/proc` as their home directory which doesn't make sense at all to check within this rule.

Reply:
I will update remediations so they are aligned with the OVAL.